cogboard-webapp/src/hooks/index.js

```diff
@@ -11,9 +11,16 @@ export const useToggle = () => {
   return [isOpened, handleOpen, handleClose];
 };
 
-export const useFormData = (data) => {
+export const useFormData = (data, validationSchema=null, onChange=false) => {
```

Comment:
Why default for a function is false? If you want falsy value, null would be better.